modules/olmv1-finding-operators-to-install.adoc

```diff
@@ -41,10 +49,18 @@ where:
 ----
 $ opm render \
   registry.redhat.io/redhat/redhat-operator-index:v{product-version} \
-  | jq -cs '[.[] | select(.schema == "olm.bundle" \
-  and (.properties[] | select(.type == "olm.csv.metadata").value.installModes[] \
-  | select(.type == "AllNamespaces" and .supported == true)) \
-  and .spec.webhookdefinitions == null) | .package] | unique[]'
+  | jq -r --arg pkg "" ' select(.schema == "olm.bundle" and (.package == $pkg or $pkg == "")) \
+  | {package:.package, name:.name, image:.image, supportsAllNamespaces: (.properties[] \
+  | select(.type == "olm.csv.metadata").value.installModes[] \
+  | select(.type == "AllNamespaces").supported == true)} ' \
+  | tee allNamespaces.json | jq -r '.image' \
+  | xargs -I '{}' -n1 -P8 bash -c ' opm render {} > $(mktemp -d -p . -t olmv1-compat-bundle-XXXXXXXX)/bundle.json ' && bash -c 'cat olmv1-compat-bundle*/*.json' \
+  | jq '{package, name, image, requiresWebhooks: (.properties[] \
+  | select(.type == "olm.bundle.object").value.data | @base64d | fromjson \
+  | select(.kind == "ClusterServiceVersion").spec.webhookdefinitions != null)}' > webhooks.json && jq -cs ' group_by(.name)[] \
+  | reduce .[] as $item ({}; . *= $item) | . *= {compatible: ((.requiresWebhooks | not) and .supportsAllNamespaces)} \
+  | {name, package, compatible} ' allNamespaces.json webhooks.json | jq -r '. \
+  | select(.compatible == true) | .package' | sort -u
```

Comment:
Happy to remove these queries and go back to the old ones. I've updated this as a separate commit, so it is easy to remove.